book/src/themes.md

```diff
@@ -265,65 +265,69 @@ These scopes are used for theming the editor interface:
-| `ui.selection`                    | For selections in the editing area                                                             |
-| `ui.selection.primary`            |                                                                                                |
-| `ui.highlight`                    | Highlighted lines in the picker preview                                                        |
-| `ui.cursorline.primary`           | The line of the primary cursor ([if cursorline is enabled][editor-section])                    |
-| `ui.cursorline.secondary`         | The lines of any other cursors ([if cursorline is enabled][editor-section])                    |
-| `ui.cursorcolumn.primary`         | The column of the primary cursor ([if cursorcolumn is enabled][editor-section])                |
-| `ui.cursorcolumn.secondary`       | The columns of any other cursors ([if cursorcolumn is enabled][editor-section])                |
-| `warning`                         | Diagnostics warning (gutter)                                                                   |
-| `error`                           | Diagnostics error (gutter)                                                                     |
-| `info`                            | Diagnostics info (gutter)                                                                      |
-| `hint`                            | Diagnostics hint (gutter)                                                                      |
-| `diagnostic`                      | Diagnostics fallback style (editing area)                                                      |
-| `diagnostic.hint`                 | Diagnostics hint (editing area)                                                                |
-| `diagnostic.info`                 | Diagnostics info (editing area)                                                                |
-| `diagnostic.warning`              | Diagnostics warning (editing area)                                                             |
-| `diagnostic.error`                | Diagnostics error (editing area)                                                               |
+| Key                                         | Notes                                                                                          |
+| ---                                         | ---                                                                                            |
+| `ui.background`                             |                                                                                                |
+| `ui.background.separator`                   | Picker separator below input line                                                              |
+| `ui.cursor`                                 |                                                                                                |
```

Comment:
Should there be a `ui.cursor.unfocused` as well for themes that use the same cursor styling for all cursors?

Reply:
I think the question comes down to: 
Should all cursors support this behavior or only the primary?

I am not sure about the answer but I could impl it that way.

Reply:
I think the behaviour should be applied to all cursors (if the theme defines the appropriate theme keys).

Reply:
Ok I impl that and for selections as well.